cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -51,4 +52,25 @@ describe("Tab management", () => {
       cy.get('[data-cy="input-civil-time"] input').clear({ force: true }).type("123");
     });
   });
+
+  const testWorker = "Pielęgniarka 2";
+  describe("Editing workers time", () => {
+    it("properly handles worker hours edition", () => {
+      cy.get('[data-cy="exit-drawer"]').click();
+      cy.get(`[data-cy="edit-worker-${testWorker}"]`).click();
+      cy.get('[data-cy="contract-time-dropdown"]').click().get('[data-cy="half"]').click();
+      cy.get(`[data-cy="btn-save-worker"]`).click();
+      cy.contains("umowa o pracę 1/2");
```

Comment:
To mi się nie podoba ponieważ nie uzależniasz tej wartości od pracownika któremu ustawiłeś etat, a tylko sprawdzasz czy w tabele jest taka wartość. 

Teraz to dobrze działa, ale jeśli my kiedyś zmienimy testowe dane (np. dodamy pracownika z etatem 1/2), to ten check utraci sens.

Więc proponuję zrobić do tego osobny `data-cy` na godzinach pracownika który będzie uzależniony od imenia (podobnie jak przycisk do edycji`.

Reply:
Zrobione.